cava/arrow-examples/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Generated SystemVerilog files, but not files in testbench/
+*.sv
+
+# Binaries
+ArrowExamplesSV
```

Comment:
This needs something like

~~~
# Generated Haskell files
 *.hs
!ArrowExamplesSV.hs
~~~

Reply:
Oops! Fixed.

Reply:
Ah! I understand now! I've removed that file now.